docs/advanced-setup.md

```diff
@@ -31,3 +31,40 @@ as the watchdog.
 
 !!! Note
     The sd_notify communication between the bot and the systemd service manager will not work if the bot runs in a Docker container.
+
+## Logging to syslog or journald
+
+On many systems the bot can be configured to send its log messages to `syslog` or `journald`. The special values for the `--logfilename` option can be used for this:
+
+* `--logfilename journald` -- send log messages to `journald`. This needs the `systemd` python package installed as the dependency. Not available on Windows.
+
+* `--logfilename syslog:<syslog_address>` -- send log messages to `syslog` server using the `<syslog_address>` as syslog address.
+
+The syslog address can be either a Unix domain socket (socket filename) or a UDP socket specification, consisting of IP address and UDP port, separated by the ':' character.
+
+So, the following are the examples of possible usages:
+
+* `--logfilename syslog:/dev/log` -- log to syslog (rsyslog) using the `/dev/log` socket, suitable for most systems.
+* `--logfilename syslog` -- same as above, the shortcut for `/dev/log`.
+* `--logfilename syslog:/var/run/syslog` -- log to syslog (rsyslog) using the `/var/run/syslog` socket. Use this on MacOS.
+* `--logfilename syslog:localhost:514` -- log to local syslog using UDP socket, if it listens on port 514.
+* `--logfilename syslog:<ip>:514` -- log to remote syslog at IP address and port 514. This may be used on Windows for remote logging to external syslog server.
+
+Log messages are send to `journald` and `syslog` with the `user` facility. So you can see them with the following commands:
+
+* `tail -f /var/log/user`, or install a comprehansive graphical viewer (for instance, 'Log File Viewer' for Ubuntu) for the `syslog` case;
+* `journalctl -f` when logging to `journald`.
```

Comment:
Personally i would add the systemd case here - where you can get "only" the bot logs by using `journalctl -f -u freqtrade.service` ... i suspect that that'll be the most common usecase for logging to journald / syslog...